qa/L0_backend_python/lifecycle/lifecycle_test.py

```diff
@@ -241,13 +241,13 @@ def test_infer_pymodel_error(self):
             initial_metrics_value,
         )
 
-    def test_grpc_strict_error_on(self):
+    def test_triton_grpc_error_error_on(self):
         model_name = "execute_grpc_error"
         shape = [2, 2]
         number_of_requests = 2
         user_data = UserData()
         triton_client = grpcclient.InferenceServerClient(f"{_tritonserver_ipaddr}:8001")
-        metadata = {"grpc_strict": "true"}
+        metadata = {"triton_grpc_error": "true"}
```

Comment:
1. Can we add docs on this feature?
2. Was there any discussion on using headers vs request parameters etc for it?

Reply:
There is a draft TEP (Was not required, I thought would be a good idea) https://docs.google.com/document/d/1TfNAMYLsPuLrduBtAKo64YqV55IWQhcn6zayc_CBY18/edit?pli=1
headers seemed logical after discussions with the SA on all possible options.
Not sure how we would do it via request params.
We need the triton_grpc_error flag set only once while the stream starts so headers seemed logica also has no backward compatibility issues.